.spelling

```diff
@@ -820,3 +820,6 @@ v6.0.
  - docs/host-powershell/README.md
 CorePsAssemblyLoadContext.cs
 sample-dotnet1
+NTLM-based
+Kerberos-based
+post-6
```

Comment:
All I know is it didn't have one before, so I didn't add it....but for you, Steve, I'll fix it. 👍